.github/workflows/release.yml

```diff
@@ -219,8 +208,6 @@ jobs:
         if: runner.os == 'macOS'
         run: |
           mkdir archive_aarch64
-          cp target/aarch64-apple-darwin/release/dora-coordinator archive_aarch64
-          cp target/aarch64-apple-darwin/release/dora-daemon archive_aarch64
           cp target/aarch64-apple-darwin/release/dora archive_aarch64/dora
           cd archive_aarch64
           zip -r ../archive_aarch64.zip .
```

Comment:
As we only distribute a single binary as artifact, we don't need to zip it. Or is there any advantage in zipping it?

Reply:
Sure it's going to make things easier.